content/docs/BestPractices/CustomTelemetry/index.md

```diff
@@ -0,0 +1,89 @@
+---
+title: "Custom Telemetry"
+tags: ["AL","Telemetry"]
+categories: ["Best Practice"]
+---
+
+_Created by Microsoft, Described by Arend-Jan Kauffmann_
+
+## Description
+With AL it is possible to emit custom telemetry signals to Azure Application Insights. There are a number of considerations that you should keep in mind when designing custom telemetry signals.
+
+- Think about it as an API
+- Naming conventions and telemetry schema
+- Candidate data for telemetry
+- How customers use telemetry
+- Privacy
+
+## Think about it as an API
+
+Customers will build analytics and monitoring solutions on top of their telemetry data.
+
+Therefore, signal must be treated as any other API
+
+- documented
+- versioned
+- discoverable
+- non-breaking
+
+## Naming conventions and telemetry schema
+
+To make it easy for the consumer of telemetry to work with the data, please
+
+- use CamelCasing. This makes all fields in Application Insights look the same (signal logged through the AL LogMessage method will have "al" prefixed to dimension names.
```

Comment:
Don't you mean PascalCasing?

camelCase <> PascalCase

Reply:
Of course... The original word was 'PamelCasing' which seems to be a typo.